templates/index.html

```diff
@@ -2,4 +2,40 @@
 
 {% block content %}
   <h1>Welcome to Canonical</h1>
+
+  <footer class="p-footer is-dark">
+    <div class="row">
+      <hr />
+    </div>
+    <div class="row">
+      <div class="col-8">
+        <p>&copy; {{ current_year }} Canonical Ltd.<br />
+          <a href="/contact-us" class="p-footer__link">Contact information</a> |
+          <a href="https://ubuntu.com/legal" class="p-footer__link">Legal information</a> |
+          <a href="https://github.com/canonical-websites/canonical.com/issues/new" class="p-footer__link">Improve this site</a>
+        </p>
+      </div>
+      <div class="col-4 u-align--right">
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item">
+              <a href="http://www.facebook.com/ubuntucloud" class="p-icon--facebook"></a>
```

Comment:
I'd expect this icon to link to Canonical's FB page but it appears it hasn't been updated since 2011...

https://www.facebook.com/pages/category/Product-Service/Canonical-125818784107695/

Reply:
If the above isn't suitable, I'd removed the FB icon.